examples/hello_world/finance/demo.py

```diff
@@ -0,0 +1,112 @@
+import time
+import random
+from collections import OrderedDict
+from maro.simulator import Env, DecisionMode
+from maro.simulator.scenarios.finance.common.common import MarketOrder, LimitOrder, StopOrder, StopLimitOrder, OrderMode, ActionType, DecisionEvent, CancelOrder, OrderDirection
+from maro.simulator.utils.common import tick_to_frame_index
+
+auto_event_mode = False
+start_tick = 10226  # 2019-01-01
+durations = 100
+max_ep = 1
+snapshot_resolution = 1
+
+env = Env(scenario="finance", topology="demo", start_tick=start_tick, durations=durations, decision_mode=DecisionMode.Joint, snapshot_resolution=snapshot_resolution)
```

Comment:
does env provide the needed information in the environment summary, such as the start/end time point

Reply:
use a more meaningful topology name, such as APPL, MSFT, etc

Reply:
not yet